src/Microsoft.AspNetCore.Server.HttpSys/Constants.cs

```diff
@@ -13,6 +13,7 @@ internal static class Constants
         internal const string Close = "close";
         internal const string Zero = "0";
         internal const string SchemeDelimiter = "://";
+        internal const int Status101SwitchingProtocols = 101;
```

Comment:
https://github.com/aspnet/HttpAbstractions/pull/748, any other status codes we want to add?